components/public-api-server/pkg/apiv1/oidc.go

```diff
@@ -247,15 +247,6 @@ func (s *OIDCService) UpdateClientConfig(ctx context.Context, req *connect.Reque
 		return nil, connect.NewError(connect.CodeInternal, fmt.Errorf("Failed to update OIDC Client Config %s", clientConfigID.String()))
 	}
 
```

Comment:
Should we also add a check in this call for `active=true`, and return an error if so?